lib/rmt/cli/mirror.rb

```diff
@@ -99,20 +102,20 @@ def mirror_repos!(mirror, repos)
       errors << _("Repository '%{repo_name}' (%{repo_id}): %{error_message}") % {
         repo_id: repo.id, repo_name: repo.name, error_message: e.message
       }
-    ensure
-      mirrored_repo_ids << repo.id
     end
-
-    [mirrored_repo_ids, errors]
   end
 
-  def handle_errors!(errors)
-    return if errors.empty?
-
-    raise RMT::CLI::Error.new(
-      _('The following errors ocurred while mirroring:%{errors_list}') % {
-        errors_list: "\n" + errors.join("\n")
-      }
-    )
+  def finish_execution
+    if errors.empty?
+      logger.info("\e[32m" + _('Mirroring complete.') + "\e[0m")
+    else
+      errors_list = errors.map { |e| e.end_with?('.') ? e : e + '.' }.join("\n")
+      raise RMT::CLI::Error.new(
```

Comment:
**This is not a PR blocking comment:**

with our error handling, the output here doesn't go to the logger.

```
I, [2020-09-07T15:13:01.040063 #11056]  INFO -- : Mirroring repository ivan_broken to /usr/share/rmt/public/repo/repositories/home:/ikapelyukhin:/dummy_repo/openSUSE_Leap_15.11/
I, [2020-09-07T15:13:01.111464 #11056]  INFO -- : Mirroring repository ivan to /usr/share/rmt/public/repo/repositories/home:/ikapelyukhin:/dummy_repo/openSUSE_Leap_15.1/
I, [2020-09-07T15:13:01.182304 #11056]  INFO -- : → repomd.xml
I, [2020-09-07T15:13:01.191599 #11056]  INFO -- : → repomd.xml.asc
I, [2020-09-07T15:13:01.196299 #11056]  INFO -- : → repomd.xml.key
I, [2020-09-07T15:13:01.271037 #11056]  INFO -- : → 44d990e1d1d6ff48ce8a59058edadd35460fc74426ed1daf28333d61cda85721-filelists.xml.gz
I, [2020-09-07T15:13:01.277293 #11056]  INFO -- : → e587bdb63d42bf5ef93f8cf941b9d297c4c8e5eaf176e6548911223d73708adf-other.xml.gz
I, [2020-09-07T15:13:01.381753 #11056]  INFO -- : ↓ 360c5ae3148c7e194348935b98bd80056f4fc991e0e865e4aa2fa1a541fa4805-primary.xml.gz
The following errors occurred while mirroring:
Repository 'ivan_broken' (1853): Error while mirroring metadata: repodata/repomd.xml - HTTP request failed with code 404.
Mirroring completed with errors.
```

This seems like a larger general issue where I can't think of an easy solution for now, so it can be fixed later.